tests/testthat/test-utils.R

```diff
@@ -100,3 +100,104 @@ test_that("build_app_title builts a valid tag", {
   testthat::expect_silent(validate_app_title_tag(valid_title_local))
   testthat::expect_silent(validate_app_title_tag(valid_title_remote))
 })
+
+
+# create_app_id ----
+testthat::test_that("create_app_id: 'data' accepts teal_data or teal_data_module", {
+  testthat::expect_no_error(create_app_id(teal.data::teal_data(), modules(example_module())))
+
+  tdm <- teal_data_module(
+    ui = function(id) div(),
+    server = function(id) NULL
+  )
+  testthat::expect_no_error(create_app_id(tdm, modules(example_module())))
+
+  testthat::expect_error(
+    create_app_id(iris, modules(example_module())),
+    "Assertion on 'data' failed: Must inherit from class 'teal_data'/'teal_data_module'"
+  )
+})
+
+testthat::test_that("create_app_id: 'modules' accepts modules", {
+  testthat::expect_no_error(create_app_id(teal.data::teal_data(), modules(example_module())))
+
+  testthat::expect_error(
+    create_app_id(teal.data::teal_data(), example_module()),
+    "Assertion on 'modules' failed: Must inherit from class 'teal_modules'"
+  )
+})
+
+testthat::test_that("create_app_id returns a character string", {
+  testthat::expect_true(
+    checkmate::test_string(
+      create_app_id(teal.data::teal_data(), modules(example_module()))
+    )
+  )
```

Comment:
All of `checkmate::{assert/check/test}_*` assertions have an equivalent `expect_*`

```suggestion
  checkmate::expect_string(
    create_app_id(teal.data::teal_data(), modules(example_module()))
  )
```

Reply:
Excellent!